server/tests/fixtures/email/emailData.js

```diff
@@ -0,0 +1,10 @@
+export default {
```

Comment:
Why is this in an array, It should be validUsers then. If it is single it should be in an object